### PR TITLE
Merge 2.x branch into master

### PR DESCRIPTION
**Pull request by igorbernstein2**
After some internal discussions, we decided that the changes in this branch don't need a major version bump

### CODE REVIEW

**Comment by igorbernstein2** [comment]
@googlebot I consent

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml [hunk 1]
```diff
@@ -148,7 +148,7 @@ limitations under the License.
                                 <phase>integration-test</phase>
                                 <configuration>
                                     <includes>
-                                        <include>**/IntegrationTests.java</include>
+                                        <include>**/TestKnownHBaseGap.java</include>
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml]: why are we changing this?

Comment (igorbernstein2) [reply bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml]: Previously we only had tests that ran against:
1. both hbase & bigtable
2. just hbase

We added a test that should only run against bigtable.

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestCachedConnections.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
+package com.google.cloud.bigtable.hbase;
```
Comment (rahulKQL) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestCachedConnections.java]: Can we please add license header here

**Comment by igorbernstein2** [comment]
Thanks for reviewing!

**Pull request merged**